crates/matrix-sdk-base/src/store/integration_tests.rs

```diff
@@ -918,25 +913,12 @@ impl StateStoreIntegrationTests for DynStateStore {
         Ok(())
     }
 
-    async fn test_persist_invited_room(&self) -> Result<()> {
```

Comment:
Are we not persisting invited rooms anymore? :sweat_smile: 
Can you use the non-deprecated method in this test, instead (or explain why it's totally fine to get rid of this test — because this is tested elsewhere)?

Reply:
To be honest, I am not sure what was the purpose of this test. It just calls `populate` then checks the number of stripped room infos, and we already did that in `test_populate_store`.

Reply:
And the reason why its fine to remove the calls to `get_stripped_room_infos` is that stripped room are included with other results in `get_room_infos`. It used to not be the case a long time ago, but when it was changed this method was deprecated and unused but we kept the tests for it.